sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultJcaProviderTest.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.springframework.util.StringUtils;
+
+import java.security.KeyStore;
+import java.security.Security;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/**
+ * The JUnit tests for the KeyVaultProvider class.
+ */
+@EnabledIfEnvironmentVariable(named = "AZURE_KEYVAULT_CERTIFICATE_NAME", matches = ".*")
+public class KeyVaultJcaProviderTest {
+
+    public static void putEnvironmentPropertyToSystemProperty(String key) {
+        Optional.of(key)
+                .map(System::getenv)
+                .filter(StringUtils::hasText)
+                .ifPresent(value -> {
+                    if (key.equals("AZURE_KEYVAULT_URI")) {
+                        System.getProperties().put(
+                            key.toLowerCase().replaceAll("_", "."), value);
+                    } else {
```

Comment:
what if the key is `AZURE_KEYVAULT`, is the logic right here?

Reply:
In each test case, it defines putEnvironmentPropertyToSystemProperty() for itself, does it really needed?

Reply:
I think change the signature from  putEnvironmentPropertyToSystemProperty(String key) to  putEnvironmentPropertyToSystemProperty(List<String> key)   is better